src/hwcrypto.js

```diff
@@ -52,7 +50,7 @@ var hwcrypto = (function hwcrypto() {
             div.setAttribute("id", 'pluginLocation' + element);
             document.body.appendChild(div);
             // Must not manipulate body's innerHTML directly, otherwise previous Element references get lost
-            document.getElementById('pluginLocation' + element).innerHTML = objectTag;
+            div.innerHTML = objectTag;
```

Comment:
You already have a reference to the element, so no need to search the DOM for it